presets/workspace/dependencies/requirements.txt

```diff
@@ -1,9 +1,9 @@
 # Dependencies for TFS
 
 # Core Dependencies
-vllm==0.6.3
-transformers == 4.45.0
-torch==2.4.0
+vllm==0.7.2
+transformers == 4.49.0
+torch==2.5.1
```

Comment:
These dependencies are shared with transformers api. Ensure inference_api UTs work

Reply:
I think the UT passes.